light-client/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "light-client"
+version = "0.1.0"
+edition = "2021"
+repository = "https://github.com/ethereum/trin"
+license = "GPL-3.0"
+readme = "README.md"
+keywords = ["ethereum", "portal-network"]
+categories = ["cryptography::cryptocurrencies"]
+description = "Beacon chain light client implementation"
+
+[dependencies]
+async-trait = "0.1.57"
+chrono = "0.4.22"
+eyre = "0.6.8"
+figment = { version = "0.10.7", features = ["toml", "env"] }
+jsonrpsee = { version = "0.15.1", features = ["full"] }
+serde = { version = "1.0.143", features = ["derive"] }
+serde_yaml = "0.9.14"
+tracing-subscriber = "0.3.15"
+hex = "0.4.3"
+reqwest = { version = "0.11.13", features = ["json"] }
+serde_json = "1.0.85"
+ssz-rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "d09f55b4f8554491e3431e01af1c32347a8781cd" }
```

Comment:
Hahaha, so much for our efforts to use a single ssz library in trin - I guess this is unavoidable though

Reply:
haha, oh noes!